CHANGELOG.rst

```diff
@@ -3,6 +3,14 @@
 0.9.2
```

Comment:
this should be 0.9.3

Reply:
@rokroskar yeah I realized I soon as I merged. I added another commit to `0.9.x` directly (this is the branch the PR was targeting) and fixed this. The `0.9.3` release targets this later commit where the changelog is correct.